pkg/scheduler/eventhandlers.go

```diff
@@ -156,61 +155,13 @@ func (sched *Scheduler) deleteNodeFromCache(obj interface{}) {
 }
 
 func (sched *Scheduler) onCSINodeAdd(obj interface{}) {
-	csiNode, ok := obj.(*storagev1beta1.CSINode)
-	if !ok {
-		klog.Errorf("cannot convert to *storagev1beta1.CSINode: %v", obj)
-		return
-	}
-
-	if err := sched.SchedulerCache.AddCSINode(csiNode); err != nil {
-		klog.Errorf("scheduler cache AddCSINode failed: %v", err)
-	}
-
 	sched.SchedulingQueue.MoveAllToActiveQueue()
 }
 
 func (sched *Scheduler) onCSINodeUpdate(oldObj, newObj interface{}) {
```

Comment:
why do we keep these stubs here now that they dont' do anything on update? If we want to just maintain `sched.SchedulingQueue.MoveAllToActiveQueue()` behavior all callers of these funcs should just call that function directly

Reply:
we need to wrap them because MoveAllToActiveQueue  has a different signature compared to the callbacks AddFunc and UpdateFunc.